### DISCUSSION TITLE
Using current license on a new on-prem server for testing

### DISCUSSION THREAD

**Discussion by conwayatmundotca** [community/community#146128] [Enterprise]
### Select Topic Area

General

### Body

Hi, 

I want to provision an new on-prem server using with the latest version for testing and possible mirgration. 

If i run a ghe-restore from backup on the test machine, are there any licensing issues with the test server using the same license as the production server.

**Comment by queenofcorgis** [comment]
@conwayatmundotca great question! 

From our [docs](https://docs.github.com/en/enterprise-server@3.15/admin/installing-your-enterprise-server/setting-up-a-github-enterprise-server-instance/setting-up-a-staging-instance):
> You may reuse your existing GitHub Enterprise license file as long as the staging instance is not used in a production capacity.
